dist/ovh-angular-actions-menu.js

```diff
@@ -77,6 +77,7 @@ angular.module("ovh-angular-actions-menu").directive("actionsMenuItem", function
  *  @param {String} [options.external=false] Is the link is an external link. In other words, will we be redirected offside of the manager ?
  *  @param {String} [options.href] The href to be redirected if item is clicked. Has no effect if state option is defined or subItems option is defined.
  *  @param {String} [options.icon] The icon class of the actions menu item icon.
+ *  @param {String} [options.svg] Template cache path of the actions menu item icon.
```

Comment:
Could you please re-run a `grunt` command to fix your dist files?
Should be: `SVG code escaped by using $sce.trustAsHtml method.` right?